brion/morphologies.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2013-2015, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef BRION_MORPHOLOGIES
+#define BRION_MORPHOLOGIES
+
+/**
+ * @file
+ * Free functions for loading sets of morphologies
+ */
+
+#include <brion/api.h>
+#include <brion/types.h>
+
+namespace brion
+{
+
+/** @namespace brion::morphologies
+ *  This namespace contains several free functions for loading sets of
+ *  morphologies.
```

Comment:
I'm starting to get my head around this change :-) Your new proposition looks better. Especially since brion::Morphology are really MorphologyReader/Writer classes, they don't load any data until told to do so. If there would be a need for this brion::morphology::load() it should be named brion::morphology::open() it seems.